### PR TITLE
Adapt Ping command to System.cmd/2

### PR DESCRIPTION
**Pull request by eddwardo**
It seems that System.cmd changed over time from System.cmd/1 to
System.cmd/2. Therefore, current state of code is not running properly.
Issue with Mac Os remains.

### CODE REVIEW

File: 09-ping.exs [hunk 1]
```diff
@@ -21,16 +21,20 @@ defmodule Ping do
   Ping a single IP address and return true if there is a response."
   """
   def ping(ip) do
-    result = System.cmd(ping_cmd(ip))
-    not Regex.match?(~r/100(\.0)?% packet loss/, result)
+    # return code should be handled somehow with pattern matching
+    {cmd_output, _} = System.cmd("ping", ping_args(ip))
+    valid_output = "1 packets transmitted, 1 packets received, 0.0% packet loss"
+    String.contains?(cmd_output, valid_output)
```
Comment (seven1m) [inline 09-ping.exs]: Why remove the regex approach? Seems it would be more platform independent and less reliant on that one output format.

**Comment by seven1m** [comment]
Looks great! With exception of the regex feedback, would merge.

**Comment by eddwardo** [comment]
Regex approach is back :)

**Comment by seven1m** [comment]
Great! Thanks!

**Pull request merged**